common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3021,6 +3021,13 @@ public static enum ConfVars {
     HIVE_ZOOKEEPER_KILLQUERY_NAMESPACE("hive.zookeeper.killquery.namespace", "killQueries",
         "When kill query coordination is enabled, uses this namespace for registering queries to kill with zookeeper"),
 
+    HIVE_GETPARTITIONS_MAX_RETRIES("hive.getpartitions.max.retries", 5,
```

Comment:
Sorry, missed this one. 
Use similar naming convention as what is used for msck repair one.
Something like  HIVE_GETPARTITIONS_BATCH_MAX_RETRIES("hive.getpartitions.batch.max.retries".......

Reply:
Done. Also i moved this property to metastoreConf since other metastore level properties are in metastoreConf.java